Makefile

```diff
@@ -18,6 +18,8 @@ KIND_CLUSTER_NAME ?= containerd-wasm
 build:
 	cargo build -p containerd-shim-wasm --features generate_bindings $(RELEASE_FLAG)
 	cargo build $(RELEASE_FLAG)
+	# no libcontainer feature on containerd-shim-wasm
+	cargo build $(RELEASE_FLAG) --no-default-features -p containerd-shim-wasm
```

Comment:
This feels backwards.  The shims that want to use the `youkiInstance` should be opting into the feature.

Reply:
Done.